frontend/src/text_resources.js

```diff
@@ -369,6 +369,14 @@ let LOCALIZE = new LocalizedStrings({
         taskContent: {
           task: "Your task(s):",
           reasonsForAction: "Your reasons for action:"
+        },
+        deleteResponseConfirmation: {
+          title: "Are you sure you want to delete this response?",
+          systemMessageTitle: "Warning!",
```

Comment:
A bit nitpicky, but I think "Warning!" would be something we would use more than once. Should this be under commons?
We could also just move it if and when we create another dialog that also needs it. I'm good with either.

Reply:
I think it's fine to have duplicated strings - it prevents us from changing it and causing problems in other places.

Reply:
Ok. I'll resolve this then

Reply:
The other system messages have their own titles, like the following:

![image](https://user-images.githubusercontent.com/23021242/56211050-13937c80-6025-11e9-9fb8-6707f5cee8d3.png)

So I'm not sure that we'll use the word 'warning' as often as we think.

Reply:
We'll keep it that way for now and if we use this word more often, then we can add it to the _commons_ words.